### DISCUSSION TITLE
Please add the ability to rewrite the urls of custom post types and custom taxonomies

### DISCUSSION THREAD

**Discussion by 2-fly-4-ai** [wpengine/faustjs#1347] [Ideas]

**Comment by theodesp** [comment]
Hey @2-fly-4-ai Thank you for the feedback. 
Could you please elaborate a bit more about this feature request? Is this supposed to be a feature for Faust of the Faust WordPress Plugin? 

Is this rewrite cannot be accomplished using [nextjs redirects system](https://nextjs.org/docs/api-reference/next.config.js/redirects)?
